### PR TITLE
fix: support href and to props for EsButton

### PR DESCRIPTION
**Pull request by hroth1994**
### 🔗 Linked issue

https://energysage.atlassian.net/browse/CED-1863

### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation or readme)
- [x] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

The new EsButton was missing `href` and `to` props. The previous implementation used Bootstrap Vue's button which has support for these props, while the new implementation uses PrimeVue's Button which does not have support for these props. If these props exist, NuxtLink is used with button styles.

### 🥼 Testing

Tested locally on Image first with button example on the [CTA card page](http://localhost:8500/organisms/cta-card) and on the [Nav bar page](http://localhost:8500/organisms/nav-bar) in the Account menu

#### 🧐 Feedback Requested / Focus Areas

- General

### 📝 Checklist

- [x] I have linked an issue or discussion.
- [x] I have updated the documentation accordingly.
- [x] I have documented testing approach

### CODE REVIEW

**Review by ericdouglaspratt — CHANGES_REQUESTED** [review]
Looks good! Just two things:

One - should we add to the EsButton docs page that EsButton is now also extended from NuxtLink, not just PrimeVue Button?

File: es-ds-docs/pages/molecules/button.vue [hunk 1]
```diff
@@ -107,11 +109,17 @@ onMounted(async () => {
         <h1>Button</h1>
         <p>
             Extended from
-            <a
+            <nuxt-link
                 href="https://v3.primevue.org/button/"
                 target="_blank">
-                primevue button
-            </a>
+                PrimeBue Button
```
Comment (ericdouglaspratt) [inline es-ds-docs/pages/molecules/button.vue:115]: Typo: "PrimeBue" --> "PrimeVue"

**Review by ericdouglaspratt — APPROVED** [review]
One typo but otherwise looks great!

File: es-ds-docs/pages/molecules/button.vue (hunk 1 above)
Comment (hroth1994) [reply es-ds-docs/pages/molecules/button.vue:115]: well that definitely isn't consistent 😅

**Pull request merged**